internal/runners/packages/search.go

```diff
@@ -71,7 +76,34 @@ func (s *Search) Run(params SearchRunParams, nstype model.NamespaceType) error {
 		)
 	}
 
-	s.out.Print(output.Prepare(formatSearchResults(packages, params.Ingredient.Namespace != ""), packages))
+	var vulns []*model.VulnerabilityIngredient
+	if s.auth.Authenticated() {
+		vulns, err = s.getVulns(packages)
+		if err != nil {
+			return errs.Wrap(err, "Could not fetch vulnerabilities")
+		}
+	}
+
+	results, err := createSearchResults(packages, vulns)
+	if err != nil {
+		return errs.Wrap(err, "Could not create search table")
+	}
+
+	if s.out.Type().IsStructured() || !s.out.Config().Interactive {
+		s.out.Print(results)
```

Comment:
For non-interactive, it seems to restrict the first column size too aggressively:

```
   Name           tensor-algebra-dont-use
   Description    bla
   License        MIT
   Versions        - 0.0.1
   Vulnerabili    <nil>
  ties
```

Notice the Vulnerabilities title being wrapped.

Also note it holds the value `<nil>`, probably needs an omitEmpty.

Reply:
The AC states that for non-interactive we just dump everything, hence the lack of processing.

Reply:
Just dump as in don't worry about scrolling. It does not mean we do not care about the output.

Reply:
I was able to fix the `omitEmpty` however the spacing seems to be in the output library. Can I propose that we rename this field to `CVEs` for now and file a follow up to address what's going on with the table output?

Reply:
Filed here: https://activestatef.atlassian.net/browse/DX-2578